### PR TITLE
chore(PDEV-2356): Apex -- 504 Gateway Timeout When Funding Account

### PR DESCRIPTION
**Pull request by kleekich**

### CODE REVIEW

File: api/v2/account/schema/moneyTransfer.req.json [hunk 1]
```diff
@@ -35,6 +35,31 @@
       "type": "string",
       "format": "YYYY-MM-DD"
     },
+    "details": {
+      "description": "Details for IRA account deposit",
+      "type": "object",
+      "properties": {
+        "accountType": {
+          "description": "Account type of where deposqit goes to",
```
Comment (prachisshah) [inline api/v2/account/schema/moneyTransfer.req.json]: "deposit"

Comment (kleekich) [reply api/v2/account/schema/moneyTransfer.req.json]: Thanks a lot!

File: api/v2/account/schema/moneyTransfer.req.json [hunk 2]
```diff
@@ -35,6 +35,31 @@
       "type": "string",
       "format": "YYYY-MM-DD"
     },
+    "details": {
+      "description": "Details for IRA account deposit",
```
Comment (jamesdaniello) [inline api/v2/account/schema/moneyTransfer.req.json]: This is supposed to be a generic key/value details map, the api docs should not mention anything about IRA and there should not be any predefined properties for the details map either.
It's fine to leave the example as it is, however.

Comment (kleekich) [reply api/v2/account/schema/moneyTransfer.req.json]: That makes sense. Thanks, James.

File: api/v2/account/schema/moneyTransfer.req.json [hunk 3]
```diff
@@ -35,6 +35,10 @@
       "type": "string",
       "format": "YYYY-MM-DD"
     },
+    "details": {
+      "description": "Details of deposit",
```
Comment (jamesdaniello) [inline api/v2/account/schema/moneyTransfer.req.json]:
```suggestion
      "description": "Any additional details necessary for this money transfer",
```

Comment (jamesdaniello) [reply api/v2/account/schema/moneyTransfer.req.json]: It can be a deposit or withdrawal, no need to specify that only deposits can have extra details

Comment (kleekich) [reply api/v2/account/schema/moneyTransfer.req.json]: Thanks!

**Comment by jamesdaniello** [comment]
I just noticed this will also affect the v10 route, so that should also have an optional `details` field added as well
`/v10/firm/{firmGuid}/account/{accountGuid}/externalTransfer`

**Pull request merged**